phpstan.neon.dist

```diff
@@ -19,3 +20,4 @@ parameters:
         - '#^PHPDoc tag @param has invalid value#'
         - '#^Parameter \#1 \$input of class AsyncAws\\[^\\]+\\ValueObject\\[^ ]+ constructor expects (array\(\)\|)?array\([^\)]+\), array\([^\)]+\) given\.#'
         - '#Method AsyncAws\\[^:]+::[^(]+\(\) should return array<[^>]+> but returns array<int(\|string)?, string>\.$#'
+        - '#PHPDoc tag @throws with type Psr\\Cache\\CacheException is not subtype of Throwable$#'
```

Comment:
It is safe to catch a non-throwable. But you cannot throw it.

https://3v4l.org/D7nk9